x-pack/filebeat/module/coredns/log/test/coredns.log-expected.json

```diff
@@ -0,0 +1,68 @@
+[
+    {
+        "@timestamp": "2019-03-06T08:55:28.903Z",
+        "coredns.dnssec_ok": false,
+        "coredns.id": "21583",
+        "coredns.query.class": "IN",
+        "coredns.query.name": "httpbin.org.cluster.local.",
+        "coredns.query.size": 43,
+        "coredns.query.type": "A",
+        "coredns.response.code": "NXDOMAIN",
+        "coredns.response.flags": [
+            "QR",
+            "RD",
+            "RA"
+        ],
+        "coredns.response.size": 136,
+        "ecs.version": "1.0.0",
+        "event.dataset": "coredns.log",
+        "event.duration": 102078,
+        "event.module": "coredns",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "INFO",
+        "log.offset": 0,
+        "log.original": "2019-03-06T08:55:28.903Z [INFO] 172.17.0.4:36413 - 21583 \"A IN httpbin.org.cluster.local. udp 43 false 512\" NXDOMAIN qr,rd,ra 136 0.000102078s",
```

Comment:
Should this go into `message` too? We are trying to always have a message if possible.

Reply:
Actually it should be log.original according to ECS. 

log.original: This is the original log message and contains the full log message before splitting it up in multiple parts.